django_freeradius/utils.py

```diff
@@ -72,9 +72,9 @@ def prefix_generate_users(prefix, n, password_length):
 
 def generate_pdf(prefix, data):
     template = get_template(BATCH_PDF_TEMPLATE)
-    html = template.render(data)
+    html = HTML(string=template.render(data), base_url=BATCH_PDF_TEMPLATE_BASE_URL)
```

Comment:
why don't you use just `STATIC_URL` and avoid adding `BATCH_PDF_TEMPLATE_BASE_URL`?

I think adding the setting `BATCH_PDF_TEMPLATE_BASE_URL` is not necessary, if you think is necessary, please explain why.